hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -296,7 +298,7 @@ public void testCreateNamespace() throws TException {
     catalog.createNamespace(namespace2, newMeta);
     Database database2 = metastoreClient.getDatabase(namespace2.toString());
     Assert.assertEquals("There no same location for db and namespace",
-        database2.getLocationUri(), hiveLocalDir);
+            hiveLocalDir, database2.getLocationUri());
```

Comment:
nit: Please fix the indentation.

Why did we change this?

Reply:
IIUC, `hiveLocalDir` should be the **expected** value, while `database2.getLocationUri()` should be the actual **value** in this assertion.